estuary/api/v1.py

```diff
@@ -240,14 +242,21 @@ def get_siblings(resource, uid):
     :raises NotFound: if the item is not found
     :raises ValidationError: if an invalid resource was requested
     """
+    story_type = request.args.get('story_type', 'container').lower()
     # This is the node that is part of a story which has the relationship to the desired
     # sibling nodes
     story_node = get_neo4j_node(resource, uid)
     if not story_node:
         raise NotFound('This item does not exist')
 
-    # Defaulting to Container Story until we modify this endpoint to suit the module changes
-    story_manager = ContainerStoryManager()
+    for class_name in current_app.config['STORY_MANAGER_SEQUENCE']:
+        if class_name.lower().startswith(story_type):
+            story_manager = getattr(estuary.utils.story, class_name)()
```

Comment:
This doesn't take into account if the config has an invalid class, but let's deal with that in a separate PR.

Reply:
I thought we agreed that since it's coming from config we don't need to deal with that

Reply:
Well we don't know if the classes in the config are valid. It's not that big of a deal because this isn't deployed by anyone but us, but it's still good practice.